src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -14,6 +14,15 @@
   <PropertyGroup Condition=" '$(TargetsBrowser)' == 'true'">
     <DefineConstants>$(DefineConstants);TARGETS_BROWSER</DefineConstants>
   </PropertyGroup>
+  <!-- ILLinker settings -->
+  <PropertyGroup>
+    <ILLinkDirectory>$(MSBuildThisFileDirectory)src\ILLink\</ILLinkDirectory>
+  </PropertyGroup>
+  <ItemGroup>
+    <ILLinkSubstitutionsXmls Include="$(ILLinkDirectory)ILLink.Substitutions.$(Platform).xml"
```

Comment:
I don't think `$(Platform)` works here. We only build per-platform binaries for System.Private.CoreLib. All other libraries are cross compiled by TFM and OS, but not by architecture.

If we use a full feature switch, this will go away as the Blazor SDK can just default the MSBuild setting to add `--feature System.Net.Http.DiagnosticsHandler.IsSupported false` when running the linker.